### DISCUSSION TITLE
Could there be copyright problems if valve finds out about this?

### DISCUSSION THREAD

**Discussion by ChaoticPumpkin** [nillerusr/source-engine#6] [Q&A]
So this appears to be using the code that was leaked in 2020 and I'm just wondering if valve has shut any other projects down or anything that uses this code? I know that valve is slightly better than other companies with fan projects but I'm just worried because this is a great project and it would be sad to see it shut down.

**Comment by fathimaognyana** [comment]
CTRL+C CTRL+Ving a reply to a similar comment I made on reddit before:

Yes, Valve has the legal right to do that. However, they wont because:

1 - It would go against the precedent they set by allowing freeuse** of the 2003 leak in fan projects.

2 - This project has no direct effect on the sales, operation of online services, nor VAC of Half Life 2 or any other source game.

4 - Its a good faith project intended to just allow people to customize and port the source engine to other platforms, so taking it down would bring bad publicity.

**Valve's exception to the free use of the 2003 leak (and any other Valve IP/Game) is if it is used for an unlicensed commercial product, such as a mod of HL2 that the devs charge people to download and install without prior approval and licensing from valve. Two examples of Licensed Games/Mods using their IPs that were approved for a commercial release by Valve are Black Mesa and HDTF.

**Comment by tifasoftware** [comment]
I honestly think that valve should use this project to publish a release for apple silicon. All the hard work is essentially done for them.

**Reply by SgtSalmon** [reply]
Yeah. They could possibly get a s*** ton of money from apple to expand their game library on mac (Apple seems to be leaning heavy into gaming, what with the RT cores and new game announcements every wwdc)
